content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

```diff
@@ -0,0 +1,264 @@
+---
+title: "Install and Manage MAAS Gateway"
+metaTitle: "Install and Manage MAAS Private Cloud Gateway"
+metaDescription: "Learn how to install and manage the MAAS Private Cloud Gateway in Palette."
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+# Overview 
+
+The Private Cloud Gateway (PCG) enables support for isolated private cloud or data center environments. You can set up the cloud gateway as a single-node or three-node cluster based on your requirements for high availability (HA).  
+<br />
+
+<InfoBox>
+
+For production environments, we recommend using three nodes. If you initially set up the gateway with one node, you can resize it at a later time. 
+
+</InfoBox>
+
+
+Palette provides an installer in the form of a Docker container that is temporarily deployed on your laptop, workstation or jump box. You can use the installer on any system that has a Docker daemon installed and connectivity to Palette and the MAAS identity endpoint.
+
+
+# Prerequisites
+
+- Canonical [MAAS installed](https://maas.io/), set up, and available in your environment.
+
+
+- A linux computer with a Docker daemon installed and a connection to Palette and the MAAS endpoint. The installer must be invoked on a linux system. We have testing the gateway installation using Ubuntu 20.04.
+
+
+- One IP address for a Kubernetes control plane.
+
+
+- One IP address for a single-node gateway or three IP addresses for a three-node gateway.
```

Comment:
@antongisli @TylerGillson 
The screenshot shows how the original doc lists it. Maybe use the bullet/sub-bullet format from the original. This answers the question above. This can be changed as follows - the bullet point about workload cluster addresses a potential missing prereq shown in the screenshot in the comment for line 46.

- PCG IP requirements as follows:
   - One IP address for a single-node gateway or three IP addresses for a three-node gateway.
   - One IP for a Kubernetes control plane.
- One IP for each workload cluster.



![CleanShot 2023-02-28 at 17 08 46](https://user-images.githubusercontent.com/117382432/222019093-38a10759-d901-4210-aa78-2a42416c329a.png)

Reply:
"One IP for each workload cluster." doesn't make sense to me either.